AutomatedTesting/Gem/PythonTests/scripting/VariableManager_Default_CreateDeleteVars.py

```diff
@@ -30,81 +38,83 @@ def VariableManager_Default_CreateDeleteVars():
-
-    # 7) Close SC window
-    general.close_pane("Script Canvas")
-
-
-if __name__ == "__main__":
-    import ImportPathHelper as imports
-
-    imports.init()
-    from utils import Report
-
-    Report.start_test(VariableManager_Default_CreateDeleteVars)
+
+        graph_vars = variable_manager.findChild(QtWidgets.QTableView, "graphVariables")
+
+        # 5) Create variable_type of each type and verify if it is created
+        for index, variable_type in enumerate(VARIABLE_TYPES):
+            # Create new variable_type
+            add_button = variable_manager.findChild(QtWidgets.QPushButton, "addButton")
+            # Click on Create Variable button and wait for it to render
+            add_button.click()
+            helper.wait_for_condition((
+                lambda: variable_manager.findChild(QtWidgets.QTableView, VARIABLE_PALETTE_QT) is not None), WAIT_TIME_3)
+
+            # Select variable_type type
+            table_view = variable_manager.findChild(QtWidgets.QTableView, VARIABLE_PALETTE_QT)
+            model_index = pyside_utils.find_child_by_pattern(table_view, variable_type)
+            # Click on it to create variable_type
+            pyside_utils.item_view_index_mouse_click(table_view, model_index)
+
+            # Drill into the variable_type list and make sure the new variable_type was created as expected
+            # Check if the number of rows in our variable_type list increased
+            row_count_increased = helper.wait_for_condition(lambda: graph_vars.model().rowCount(QtCore.QModelIndex()) == (
+                index + 1), WAIT_TIME_3)
+            # Get the variable_type entry from the list
+            new_var_entry = pyside_utils.find_child_by_pattern(graph_vars, f"Variable {index+1}")
+            # Get the variable_type's type
+            new_var_type = new_var_entry.siblingAtColumn(1).data(QtCore.Qt.DisplayRole)
+
+            # Verify the list size, list entry and variable_type type all match expectation
+            result = row_count_increased and (new_var_entry is not None) and (new_var_type == variable_type)
+
+            test_output = self.generate_variable_test_output(variable_type, "create")
+            Report.info(f"{test_output}: {result}")
+
+        # 6) Delete each type of variable_type and verify if it is deleted
+        for index, variable_type in enumerate(VARIABLE_TYPES):
+            # Get the variable entry from the list
+            var_entry = pyside_utils.find_child_by_pattern(graph_vars, f"Variable {index+1}")
+            # Click on it
+            pyside_utils.item_view_index_mouse_click(graph_vars, var_entry)
+            # Press the delete key
+            QtTest.QTest.keyClick(graph_vars, QtCore.Qt.Key_Delete, QtCore.Qt.NoModifier)
+            # Check the number of rows in the variable list
+            result = graph_vars.model().rowCount(QtCore.QModelIndex()) == (len(VARIABLE_TYPES) - (index + 1))
```

Comment:
We don't need a line-by-line commenting pattern. This can be summarized in the original comment (it's ok to extend the comment by another line to provide a flow summary if desired).

Reply:
removed or consolidated many of these comments. Want to preserve the comments around the lines where one of the bug fixes is though since that step-by-step process was a bit confusing when I stepped into this script.